src/main/resources/db/migration/changes/Reviews.json

```
@@ -0,0 +1,109 @@
+{ "databaseChangeLog": [
+    {
+        "changeSet": {
+          "id": "Reviews-6",
+          "author": "KEVINY",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "REVIEWS"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                        "autoIncrement": true,
+                        "constraints": {
+                            "primaryKey": true,
+                            "primaryKeyName": "REVIEWS_PK"
+                        },
+                          "name": "ID",
+                          "type": "BIGINT"
+                        }
+                  },
+                  {
+                    "column": {
+                      "constraints": {
+                        "nullable": false
+                      },
+                      "name": "STUDENT_ID",
+                      "type": "INTEGER"
+                    }
+                  },
+                  {
+                    "column": {
+                      "constraints": {
+                        "nullable": false
+                      },
+                      "name": "ITEM_ID",
+                      "type": "INTEGER"
+                    }
+                  },
+                  {
+                    "column": {
+                      "constraints": {
+                        "nullable": false
+                      },
+                      "name": "DATE_SERVED",
+                      "type": "VARCHAR(255)"
+                    }
+                  },
+                  {
+                    "column": {
+                      "constraints":{
+                        "nullable": false,
+                        "default": "Awaiting Moderation"
+                      },
+                      "name": "STATUS",
+                      "type": "VARCHAR(255)"
+                    }
+                  },
+                  {
+                    "column": {
+                      "constraints":{
+                        "default": null
+                      },
+                      "name": "USERID",
```

```suggestion
                      "name": "USER_ID",
```

Is this why you put the `@Column` annotation in the other spot?  If so, it would be better to use the snake case convention here. You'll probably run into fewer problems down the road.

---

There was an issue with the repository function "getByUserId". If the field name was "user_id", the function getByUser_id(x) wouldn't work because this can't recognize the underscore and instead will only look for the substring "user". The only solution that worked for us was to change this field to userId even though it doesn't follow the naming convention because it allows getByUserId(x) to work.